src/Fight.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import Fight from './Fight';
+
+test('should render a button', () => {
+  render(<Fight />);
+  const linkElement = screen.getByText(/Unicorn grazes/i);
+  expect(linkElement).toBeInTheDocument();
+});
```

Comment:
Same notes as the other test, I think another good test to do is to look for the "sorry the zoo is closed" text then have the test user click the open button and then look for "the zoo is open!" text. That's a good user interactions test.